ibis/impala/tests/test_client.py

```diff
@@ -25,6 +25,15 @@ def db(con, test_data_db):
     return con.database(test_data_db)
 
 
+def test_kerberos_deps_installed(con_kerberos_no_hdfs):
+    expected_exception_type = ModuleNotFoundError
+    try:
+        _ = con_kerberos_no_hdfs.table('tpch_lineitem')
```

Comment:
All this should be the same as:
```python
with pytest.raises(ModuleNotFoundError):
    con_kerberos_no_hdfs.table('tpch_lineitem')
```

Reply:
Okay, got it! Looks like the test is failing now in Azure, because the pipeline uses conda to install ibis and conda does not yet seem to support installing optional dependencies the way pip does e.g. 
`pip install impyla[kerberos]` vs `conda install impyla[kerberos]`
So, for now I'm going to push a commit with an explicit dependency on thrift_sasl and I'm working on a PR to the Impyla feedstock that will explicitly package thrift_sasl as a default.